ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.14.yaml

```diff
@@ -164,6 +229,68 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-hypershift
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-mce-guest
+- as: aws-ipi-private-shared-vpc-phz-sts-f28
```

Comment:
`aws-ipi-private-shared-vpc-phz-sts-f28`, `aws-ipi-shared-vpc-phz-f28` and `aws-ipi-shared-vpc-phz-sts-fips-f28` cover similar profiles, I think we can keep `aws-ipi-private-shared-vpc-phz-sts-f28` and remove the other 2.